packages/optimizely-sdk/lib/core/odp/lru_cache/LRUCacheElement.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2022, Optimizely
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * LRUCacheElement represents an individual generic item within the LRUCache
+ */
+export class LRUCacheElement<V> {
```

Comment:
```suggestion
export class CacheElement<V> {
```
This element is a generic cache element, not only for LRU.